Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Basic docker image for Monocle
+# Usage:
+#   docker build -t Monocle
+#   docker run -d --name Monocle -P Monocle
+
+FROM python:3.6
+
+# Working directory for the application
+WORKDIR /usr/src/app
+
+# Set Entrypoint with hard-coded options
+ENTRYPOINT ["python3", "./scan.py"]
```

Comment:
I would like to suggest to change to `ENTRYPOINT ["python3"]` and add `CMD ["./scan.py"]`, so we could start a web instance only changing the command, not the whole entrypoint.